agenta-backend/agenta_backend/models/api/api_models.py

```diff
@@ -95,6 +95,13 @@ class AppVariantFromImagePayload(BaseModel):
     variant_name: str
 
 
+class ModifiedBy(BaseModel):
```

Comment:
We'll be duplicating username and profile picture (personal data) all over the app.
Let's avoid this.

Reply:
Applicable wherever this happens.